src/inputs/ObjectSearch.tsx

```diff
@@ -136,6 +137,8 @@ class ObjectSearch extends Component<IObjectSearchProps> {
       ;
 
     this.search = value;
+    if (!searchOnEmpty && !this.hasSearch) { return; }
```

Comment:
This makes it so if you type and delete, it continues to show the results from the typed search if `!searchOnEmpty`. It's minor but assumedly unintentional?

Reply:
:( yeah I'll work on that